### PR TITLE
Set identity logic

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

Implemented logic for the following
- account unification, together with set identity (those two can't be tested as a single batch atm because local node with accounts palled doesn't have identity pallet. Separate tests worked for me)
- NFT repository, to fetch tokens owned by an user

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

- (ex: Add feature A)

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

> \*Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

File: src/v2/repositories/implementations/BluezNftRepository.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// TODO move to Token API
+import axios from 'axios';
+import { injectable } from 'inversify';
+import { NftMetadata } from 'src/v2/models';
+import { INftRepository } from 'src/v2/repositories';
+
+@injectable()
+export class BluezNftRepository implements INftRepository {
+  private readonly baseUrl = 'https://api.bluez.app/api/nft/v3/API_KEY';
```
Comment (gluneau) [inline src/v2/repositories/implementations/BluezNftRepository.ts:9]: API Key seems required for this "Set Identity"

File: src/components/assets/Assets.vue [hunk 2]
```diff
@@ -5,6 +5,7 @@
     </div>
     <div class="container--assets">
       <astar-button @click="unify()">Unify accounts</astar-button>
+      <astar-button @click="setIdentity()">Set setIdentity</astar-button>
```
Comment (gluneau) [inline src/components/assets/Assets.vue:8]: An extra 'set'

**Review by gluneau — CHANGES_REQUESTED** [review]
Few changes, but I get an error while clicking on Unify accounts on all networks.
image

**Comment by bobo-k2** [comment]
> Few changes, but I get an error while clicking on Unify accounts on all networks.
> 
> image
> 
> 

Unfortunately you can't test the code from UI yet, because we don't have all pieces connected yet

File: src/components/assets/Assets.vue (hunk 2 above)
Comment (bobo-k2) [reply src/components/assets/Assets.vue:8]: Those are temporary buttons so I can test pieces of logic

**Review by gluneau — APPROVED** [review]
Will follow changes in account unification branch/PR

**Pull request merged**